aws/table_aws_docdb_cluster_instance.go

```diff
@@ -0,0 +1,383 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/docdb"
+	"github.com/aws/aws-sdk-go-v2/service/docdb/types"
+
+	docdbv1 "github.com/aws/aws-sdk-go/service/docdb"
+
+	"github.com/turbot/go-kit/helpers"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsDocDBClusterInstance(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_docdb_cluster_instance",
+		Description: "AWS DocumentDB Cluster Instance",
```

Comment:
@ParthaI The error `InvalidParameterCombination` used to come while using `Filter` input and `DBInstanceIdentifier` input together, which isn't being used right now. Sorry for the confusion.
Have removed `InvalidParameterCombination` from the list.

Also, `Getconfig` won't be required now since `ListConfig` works as expected and suffice the requirement.

Reply:
Thanks @karanpopat for the clarification. 👍